Automator/gui/main.py

```diff
@@ -48,3 +56,27 @@ def __init__(self, *args, **kwargs):
         self.setWindowTitle('24HS-Automator')
         self.setMinimumSize(500, 300)
         self.setCentralWidget(main_widget)
+        
+        try:
+            import requests
+            response = requests.get("https://api.github.com/repos/24HourSupport/Automator/releases/latest", timeout=3)
+            Latest_version = response.json()["name"].replace('v', '').strip()
+
+            if (Current_version < Latest_version):
+                UpToDateBox(self).exec()
+        except:
```

Comment:
```suggestion
        except requests.exceptions.ConnectTimeout:
```

Reply:
Why only handle this specific type of exception? Checking for updates should never be a fatal error here.